sections/03_categorize.md

```diff
@@ -28,7 +28,76 @@ we share these data while respecting privacy? I've made a section for each of
 these. Feel free to add more. I see each section as something on the order of
 1-2 paragraphs in our context.*
 
-#### Standardization/integration
+#### Clinical Care
+
+#### Imaging Modalities (static + dynamic)
+
+* Imaging #163, #164
+
+#### Electronic Health Records
+
+EHR data include substantial amounts of free text, which remains challenging to
+approach [@doi:10.1136/amiajnl-2011-000501]. Often, researchers developing
+algorithms that perform well on specific tasks must design and implement
+domain-specific features [@doi:10.1136/amiajnl-2011-000150]. These features
+capture unique aspects of the literature being processed. Deep learning methods
+are natural feature constructors. In recent work, the authors evaluated the
+extent to which deep learning methods could be applied on top of generic
+features for domain-specific concept extraction [@arxiv:1611.08373]. They found
+that performance was in line with, but did not exceed, existing state of the art
+methods. The deep learning method had performance lower than the best performing
+domain-specific method in their evaluation[@arxiv:1611.08373]. This highlights
+the challenge of predicting the eventual impact of deep learning on the field.
+This provides support that deep learning may impact the field by reducing the
+researcher time and cost required to develop specific solutions, but it may not
+lead to performance increases.
+
+TODO: survival analysis/readmission prediction methods from EHR/EMR style data
```

Comment:
This sounds good, I have nothing more to add, except for possibly mentioning other non-linear methods.  I can think of Random Forest used in survival analysis (@arxivL0811.1645)